### ISSUE TITLE
As a CIF user, I want to see the type of my file attachment in a readable format, so that I understand which file type I am dealing with 

### ISSUE THREAD

**Issue by pbastia** [bcgov/cas-cif#481]
#### Description:

Currently file types print the mime-type, for example `application/vnd.openxmlformats-officedocument.wordprocessingml.document`

We would want to show something like "Word Document (.docx)" instead

#### Acceptance Criteria:

Given I am a CIF user
When I look at the list of attachments for a project
Then I can see the type of each file in a readable format

#### Dev notes:
This could be stored on the database and resolved by the server when the file is uploaded,
Or inferred from the mime-type on the frontend with for example https://www.npmjs.com/package/mime-names

#### Development Checklist:
- [ ] the `AttachmentTableRow` component shows the right value (strategy to be explored)
- [ ] Meets the DOD

#### Notes:

-

**Comment by LindsayMacfarlane** [comment]
@pbastia - is this ticket addressed through #611? Maybe not directly, but when testing #611, I noticed that file type is already included.

**Comment by pbastia** [comment]
Not quite, right now we display the mime type, which can be quite cryptic (for example, `application/vnd.openxmlformats-officedocument.wordprocessingml.document` for .docx files)
